packages/lab/src/alert/Alert.tsx

```diff
@@ -9,15 +9,21 @@ import styled, { css, SimpleInterpolation } from 'styled-components';
 
 function colorMixin(
   textColor: Color,
+  iconColor: Color,
   backgroundColor: Color,
   buttonHoverColor: Color,
 ): readonly SimpleInterpolation[] {
   return css`
     color: ${textColor};
+    border-color: ${textColor};
     background-color: ${backgroundColor};
 
     & .MuiAlert-icon {
-      color: ${textColor};
+      color: ${iconColor};
+    }
+
+    & .MuiAlert-action {
+      color: ${iconColor};
     }
 
     & .MuiAlert-action {
```

Comment:
@normurod72 Can we merge these 2 definitions? And we will have the text color the same as the icon color?